middleware/access_log.go

```diff
@@ -366,3 +372,28 @@ func (w *bodyDumpResponseWriter) Flush() {
 func (w *bodyDumpResponseWriter) Hijack() (net.Conn, *bufio.ReadWriter, error) {
 	return w.ResponseWriter.(http.Hijacker).Hijack()
 }
+
+func MaskSensitiveInfo(reqBody []byte, maskedParams []string) ([]byte, error) {
+	var unmarshaledRequest = make(map[string]interface{})
+	err := json.Unmarshal(reqBody, &unmarshaledRequest)
+	if err != nil {
+		fmt.Println("err", err)
+		return reqBody, err
+	}
+	for key, value := range unmarshaledRequest {
+		if contains(maskedParams, key) {
```

Comment:
use this instead
[https://github.com/retail-ai-inc/bean/blob/develop/string/str.go#L204](url)